### DISCUSSION TITLE
is that possible to customize the "map_index" or similar in dynamic mapped task

### DISCUSSION THREAD

**Discussion by Haoxing** [apache/airflow#27002] [Q&A]
Hi guys,

It is really helpful for me that you bring in the feature dynamic  task mapping since 2.3.0. Many thanks for your great efforts. There is one thing i want to ask, is that possible to customize the "map_index" or add a user-defined column for identifying the mapped task shown in the GUI? I just wonder this may be a good way to differentiate the dynamically populated tasks and more convenient for end user to quickly navigate to corresponding task. 

<img width="758" alt="image" src="https://user-images.githubusercontent.com/4607996/195242044-045691d3-a3cd-47f4-b917-38299a2538c2.png">

**Comment by potiuk** [comment]
Not yet. But there are discussions about it. Maybe you can propose a PR for that ? That's always the best way to move things forward. One of the "challenges" is that it has to work for all the ways that dynamic tasks can be mapped (there are a number of ways) - feel free to propose something or dicsuss it in the devlist.
